src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/StatusCodes.cs

```diff
@@ -216,5 +216,144 @@ public static ReadOnlySpan<byte> ToStatusBytes(int statusCode)
+                    return "409";
+                case (int)HttpStatusCode.Gone:
+                    return "410";
+                case (int)HttpStatusCode.LengthRequired:
+                    return "411";
+                case (int)HttpStatusCode.PreconditionFailed:
+                    return "412";
+                case (int)HttpStatusCode.RequestEntityTooLarge:
+                    return "413";
+                case (int)HttpStatusCode.RequestUriTooLong:
+                    return "414";
+                case (int)HttpStatusCode.UnsupportedMediaType:
+                    return "415";
+                case (int)HttpStatusCode.RequestedRangeNotSatisfiable:
+                    return "416";
+                case (int)HttpStatusCode.ExpectationFailed:
+                    return "417";
+                case (int)418:
+                    return "418";
+                case (int)419:
+                    return "419";
+                case (int)HttpStatusCode.MisdirectedRequest:
+                    return "421";
+                case (int)HttpStatusCode.UnprocessableEntity:
+                    return "422";
+                case (int)HttpStatusCode.Locked:
+                    return "423";
+                case (int)HttpStatusCode.FailedDependency:
+                    return "424";
+                case (int)HttpStatusCode.UpgradeRequired:
+                    return "426";
+                case (int)HttpStatusCode.PreconditionRequired:
+                    return "428";
+                case (int)HttpStatusCode.TooManyRequests:
+                    return "429";
+                case (int)HttpStatusCode.RequestHeaderFieldsTooLarge:
+                    return "431";
+                case (int)HttpStatusCode.UnavailableForLegalReasons:
+                    return "451";
+
+                case (int)HttpStatusCode.InternalServerError:
+                    return "500";
+                case (int)HttpStatusCode.NotImplemented:
+                    return "501";
+                case (int)HttpStatusCode.BadGateway:
+                    return "502";
+                case (int)HttpStatusCode.ServiceUnavailable:
+                    return "503";
+                case (int)HttpStatusCode.GatewayTimeout:
+                    return "504";
+                case (int)HttpStatusCode.HttpVersionNotSupported:
+                    return "505";
+                case (int)HttpStatusCode.VariantAlsoNegotiates:
+                    return "506";
+                case (int)HttpStatusCode.InsufficientStorage:
+                    return "507";
+                case (int)HttpStatusCode.LoopDetected:
+                    return "508";
+                case (int)HttpStatusCode.NotExtended:
+                    return "510";
+                case (int)HttpStatusCode.NetworkAuthenticationRequired:
+                    return "511";
+
+                default:
+                    return statusCode.ToString(CultureInfo.InvariantCulture);
```

Comment:
Would there be any difference if we just do:

```c#
public static string ToStatusString(int statusCode) => statusCode.ToString(CultureInfo.InvariantCulture);
```

Reply:
That allocates a new string.